source/sass/components/article-page.scss

```diff
@@ -56,16 +52,20 @@
 }
 
 .article-summary-link {
+  display: block;
   --spacing: 10px;
 
   margin-top: 17px;
   flex-basis: calc(50% - var(--spacing));
   flex-grow: 0;
   flex-shrink: 0;
-
   @include hover-focus-active {
     color: inherit;
   }
+
```

Comment:
if there are less than 5 links, the article summary links display block one after the other.

However if there are more than 5 and the two-col class is assigned, they display inline-block to stay aligned with one another in the two column format.